machine.go

```diff
@@ -669,6 +691,7 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 			{Target: "/usr/sbin", Link: "/sbin", Perm: 0755},
 			{Target: "/usr/bin", Link: "/bin", Perm: 0755},
 			{Target: "/usr/lib", Link: "/lib", Perm: 0755},
+			{Target: "/usr/lib64", Link: "/lib64", Perm: 0755},
```

Comment:
It would be worth testing manually on arch and reporting in a separate issue if you find there are problems. We have a CI test (which passed) but it may not cover your use-case?